draft-ietf-rats-corim.md

```diff
@@ -2799,17 +2797,17 @@ Published specification:
 
 Applications that use this media type:
 : Attestation Verifiers, Endorsers and Reference-Value providers that need to
-  transfer unprotected CoRIM payloads over HTTP(S), CoAP(S), and other
+  transfer protected CoRIM payloads over HTTP(S), CoAP(S), and other
   transports.
 
 Fragment identifier considerations:
 : n/a
 
 Magic number(s):
-: `D9 01 F5`, `D9 01 F4 D9 01 F5`
+: `D2` (for COSE), and TBD given CoAP C-F assignment with TN transformation {{?RFC9277}} (e.g., for C-F 141 `D9 63 74 01 8F D2`).
```

Comment:
There is no TN here, it's just the tagged COSE Sign1:

```suggestion
: `D2 84`
```

Reply:
I thought you wanted the +cose CoAP C-F number to serialize the .corim at rest. The COSE_Sign1-only form is just for on-the-wire.